README.md

```diff
@@ -10,3 +11,29 @@ After this, you can access the server at localhost:9000.
 To launch postgres and adminer along with play app, run `./bin/run-dev`
 This will start 3 individual containers.
 You can access play app at localhost:9000 as before, and adminer at localhost:8080.
+
+## Development standards
+
+### Client-server concerns
+
+The client should be as simple as is practical to implement the desired user experience.
+
+### Java code
+
+Java code should conform to the Google Java [styleguide](https://google.github.io/styleguide/javaguide.html). The project makes use of a linter and autoformatter for Java to help with this.
+
+Prefer using immutable collection types provided by [Guava](https://github.com/google/guava) ([API docs](https://guava.dev/releases/snapshot/api/docs/)) over the Java standard library's mutable collections unless impractical. Include a comment justifying the use of a mutable collection if you use one.
+
+#### Separation of concerns
+
+In lieu of a microservice architecture, this project necessitates special care in ensuring separation of concerns. While Play provides some structure in this regard, it should be viewed as a starting point.
+
+Code in *Play controllers* should be limited to brokering interaction between the server's business logic systems and HTTP. Code in controller's should never directly implement business logic concerns and should instead delegate to classes specific to those purposes. One way to help think about this when writing controller code: if you're writing an HTTP handler that responds with HTML, factor out business logic classes so that implementing another handler that performs the same logic but responds with JSON benefits from high code re-use.
```

Comment:
I'd consider a link to your favorite primer on MVC - I'm a bit out of date on that, honestly.

Reply:
Hmm I'll have to look around for one. If I can't find something that fits the bill I can give a quick talk like you did for Docker.